pages/urls/api/__init__.py

```diff
@@ -25,10 +25,11 @@
 '''API URLs for the pages application'''
 
 from ...api.elements import (PageElementEditableListAPIView,
-    PageElementIndexAPIView)
+    PageElementIndexAPIView, ImportDocxView)
 from ...compat import include, path
 
 urlpatterns = [
+    path('import-docx', ImportDocxView.as_view(), name='import_docx'),
```

Comment:
This URL path should most likely be under `content/editables/` (i.e. in `ages.urls.api.editables`) so it is easier to set permissions rules for all editable content.
Most likely, the `PageElement` slug should be part of the URL path (ex: r`^import/(?P<path>%s)$`) to be consistent with other APIs.